src/SharpSportsMobileLink.tsx

```diff
@@ -62,14 +63,14 @@ class SharpSportsMobileLink extends React.Component<Props> {
 }
 
 const fetchIntegration = (props: Props) => {
-    const { internalId, token } = props;
+    const { internalId, token, logoUrl } = props;
     props.onLoading?.();
     postContext('https://api.sharpsports.io/v1/context', {internalId: internalId}, token)
     .then(data => {
         props.onLoadingDismiss?.();
         props.presentWebView(
-            <WebView 
-              source={{uri: `https://ui.sharpsports.io/link/${data.cid}`}} 
+            <WebView
+              source={{uri: `https://ui.sharpsports.io/link/${data.cid}?user_logo=${logoUrl}`}}
```

Comment:
what happens if the `logoUrl` is undefined?

Reply:
just tried it. if logoUrl isn't defined, it results in a placeholder "?" in the webview.

try this:

```javascript
const buildURL = (data: any, logoUrl: string) => {
    return logoUrl ? `https://ui.sharpsports.io/link/${data.cid}?user_logo=${logoUrl}` : `https://ui.sharpsports.io/link/${data.cid}`
}
```

then call it like this

```javascript
<WebView
    source={{uri: buildURL(data, logoUrl)}}
    style={{justifyContent: "center"}}
    onNavigationStateChange={ (newNavState: WebViewNavigation) =>
        handleWebViewNavigationStateChange(props, newNavState)
    }
/>
```